db/migrate/20221104183518_create_orders.rb

```diff
@@ -0,0 +1,12 @@
+class CreateOrders < ActiveRecord::Migration[7.0]
+  def change
+    create_table :orders do |t|
+      t.string :name
+      t.string :email
+      t.string :phone
+      t.string :address
+      t.string :status
+      t.timestamps
+    end
```

Comment:
Well done so far, for this model we agreed to have an order_number which will be generated randomly, Kindly add the order_number column which will be a string, and make the association as well because the order must belong to a user and food. the status column could be an integer because we will be updating it by using the Enum method as well. don't forget to add the fault value for the status witch will be 0 that means on "pending"